index.html

```diff
@@ -11,72 +11,77 @@
-    <p class="lead">Learn how to call algorithms in the language of your choice via our API's:</p>
-    <a href="{{ site.baseurl }}/clients" class="btn btn-primary btn-raised">Client Guides</a>
-  </div>
-  <div class="syn-col syn-col-md-7">
-    <div class="syn-code-block">
-<pre><code>import Algorithmia
-input = "YOUR_USERNAME"
-client = Algorithmia.client("YOUR_API_KEY")
-algo = client.algo("demo/Hello/")
-print(algo.pipe(input))</code></pre>
+  <a href="{{ site.baseurl }}/clients" class="syn-card outlined actionable syn-link-no-decoration syn-mb-32">
+    <h3>Client Guides</h3>
+    <div class="syn-body-1 syn-text-secondary syn-mb-32">
+      Learn how to call algorithms in the language of your choice via our API's.
     </div>
-  </div>
-  <!--For smaller screens-->
-  <div class="syn-col syn-col-sm-12 syn-visible-sm">
-    <p class="lead">Learn how to call algorithms in the language of your choice via our API's:</p>
-    <a href="{{ site.baseurl }}/clients" class="btn btn-primary btn-raised">Client Guides</a>
-  </div>
-</div>
+    <div class="syn-h6 syn-mb-4 syn-text-blue syn-flex align-center">
+      View the Client Guides
+      <i class="material-icons syn-ml-8">arrow_forward</i>
+    </div>
+  </a>
 
-<div class="syn-row syn-mb-64">
-  <div class="syn-col syn-col-md-5">
-    <img src="{{ site.cdnurl }}{{ site.baseurl }}/images/sample_apps.jpg" alt="Contribute" class="syn-image-responsive">
-  </div>
-  <div class="syn-col syn-col-md-7">
-    <p class="lead">Contribute to the marketplace by adding your own algorithms:</p>
-    <a href="{{ site.baseurl }}/algorithm-development" class="btn btn-primary btn-raised">Create Algorithms</a>
-  </div>
-</div>
+  <a href="{{ site.baseurl }}/algorithm-development" class="syn-card outlined actionable syn-link-no-decoration syn-mb-32">
+    <h3>Contribute</h3>
+    <div class="syn-body-1 syn-text-secondary syn-mb-32">
+      Contribute to the marketplace by adding your own algorithms.
+    </div>
+    <div class="syn-h6 syn-mb-4 syn-text-blue syn-flex align-center">
+      Adding Your Algorithms
+      <i class="material-icons syn-ml-8">arrow_forward</i>
+    </div>
+  </a>
 
-<div class="syn-row syn-mb-64">
-  <div class="syn-col syn-col-md-5">
-    <p class="lead syn-hidden-sm">Connect to Dropbox, Amazon S3, or Algorithmia's Hosted Data:</p>
-    <a href="{{ site.baseurl }}/data" class="btn btn-primary btn-raised hidden-xs hidden-sm">Data Source Guides</a>
-  </div>
-  <div class="syn-col syn-col-md-7">
-    <img src="{{ site.cdnurl }}{{ site.baseurl }}/images/test_smallest_data_icons.jpg" alt="Sample Apps" class="syn-image-responsive">
-  </div>
-  <div class="syn-col syn-col-sm-12 syn-visible-sm">
-    <p class="lead">Connect to your Data</p>
-    <a href="{{ site.baseurl }}/data" class="btn btn-primary btn-raised">Data Source Guides</a>
-  </div>
+  <a href="{{ site.baseurl }}/algorithm-development" class="syn-card outlined actionable syn-link-no-decoration syn-mb-32">
```

Comment:
This should be `{{ site.baseurl }}/data`.

Reply:
Thanks for catching! Updated!